lib/tbd/geo.rb

```diff
@@ -352,6 +352,7 @@ def properties(surface = nil, argh = {})
     surf[:n          ] = n
     surf[:gross      ] = surface.grossArea
     surf[:filmRSI    ] = surface.filmResistance
+    surf[:spandrel   ] = spandrel?(surface)
```

Comment:
New OSut method `spandrel?` : relies first on _AdditionalProperties_ "spandrel" (true/false), or else if surface identifier includes "spandrel" string. There's no other way to distinguish spandrel panels from non-spandrel (opaque) surfaces in OpenStudio.